Corona-Warn-App/src/main/res/layout/fragment_submission_intro.xml

```diff
@@ -1,58 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools">
-
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/submission_intro_root"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:contentDescription="@string/submission_intro_accessibility_title"
-        android:fillViewport="true"
-        android:focusable="true"
-        tools:context=".ui.submission.fragment.SubmissionIntroFragment">
-
-        <include
-            android:id="@+id/submission_intro_header"
-            layout="@layout/include_header"
-            android:layout_width="@dimen/match_constraint"
-            android:layout_height="wrap_content"
-            app:icon="@{@drawable/ic_close}"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:title="@{@string/submission_intro_title}" />
```

Comment:
Those strings could be removed as well

Reply:
Done

Reply:
Got an Error while building locally 
_fragment_submission_test_result_available.xml:83: error: resource string/submission_intro_button_next_